mpas_analysis/shared/mpas_xarray/mpas_xarray.py

```diff
@@ -244,6 +244,12 @@ def preprocess(ds, calendar, simulationStartTime, timeVariableName,
     and Xylar Asay-Davis
     """
 
+    # following a suggestion by @rabernat
+    # https://github.com/pydata/xarray/issues/2064#issuecomment-381717472
+    concat_dim = 'Time'
```

Comment:
Are we 100% sure we want to hard-code this?  I'm think it would be better as an optional argument, not hard coded below.